enterprise/api/v1/serializers.py

```diff
@@ -562,6 +573,10 @@ def create(self, validated_data):
                 else:
                     enterprise_customer_user.unenroll(course_run_id)
             except (CourseEnrollmentDowngradeError, CourseEnrollmentPermissionError, HttpClientError) as exc:
+                error_message = (
+                    '[Enterprise API] An exception occurred while enrolling the user. Message: {}.'
```

Comment:
Let's also add `enterprise_customer`, `lms_user`, `tpa_user`, `user_email`, `course_run_id`, `course_mode` in this log

Reply:
Done